packages/blocks/src/_common/components/file-drop-manager.ts

```diff
@@ -53,9 +48,51 @@ export class FileDropManager {
       );
       document.body.appendChild(this._indicator);
     }
+
+    this.onDragOver = this.onDragOver.bind(this);
```

Comment:
It's recommended in our codebase to use `someMethod = () => {  ... }` to avoid using `.bind`.